### PR TITLE
INTLY-7044-added-installation-type-to-solution-explorer-cr

### PR DESCRIPTION
**Pull request by austinmartinh**
JIRA: https://issues.redhat.com/browse/INTLY-7044

## Verification
- Clone this repo
- Run the operator against a PDS/OSD cluster
- Verify the `INSTALLATION_TYPE` parameter is part of the `Spec.Parameters` field of the WebApp CR

### CODE REVIEW

File: pkg/products/solutionexplorer/reconciler.go [hunk 1]
```diff
@@ -57,6 +57,7 @@ const (
 	defaultRouteName          = "solution-explorer"
 	manifestPackage           = "integreatly-solution-explorer"
 	paramRoutingSubdomain     = "ROUTING_SUBDOMAIN"
+	paramInstallationType = "INSTALLATION TYPE"
```
Comment (dimitraz) [inline pkg/products/solutionexplorer/reconciler.go:60]: Missing an underscore
```suggestion
	paramInstallationType = "INSTALLATION_TYPE"
```

File: pkg/products/solutionexplorer/reconciler.go [hunk 2]
```diff
@@ -380,6 +381,7 @@ func (r *Reconciler) ReconcileCustomResource(ctx context.Context, installation *
 			paramIntegreatlyVersion:   version.IntegreatlyVersion,
 			paramWalkthroughLocations: defaultWalkthroughsLoc,
 			paramRoutingSubdomain:     installation.Spec.RoutingSubdomain,
+			paramInstallationType: installation.Spec.Type,
```
Comment (dimitraz) [inline pkg/products/solutionexplorer/reconciler.go:384]: Looks like you need to run a `make code/fix` to format this file

**Comment by aidenkeating** [comment]
/lgtm

Verified against OSD

Screenshot 2020-05-01 at 16 15 03

**Pull request merged**